framework/physics/physics_namespace.h

```diff
@@ -13,8 +13,7 @@ enum class OperationType
 {
   SINGLE_VALUE = 0,
   FROM_ARRAY = 1,
-  SIMPLEXS0 = 20,
-  SIMPLEXS1 = 21,
+  MODEL1G = 20,
```

Comment:
How about something like
```suggestion
  SIMPLE_ONE_GROUP = 20,
```
or
```suggestion
  SIMPLE_1G = 20,
```
What does model mean here?

Reply:
"simple" is not that important. I would say that the isotropic nature of scattering allowed in that simple model is a more important information to convey. How about `ISOTROPIC_ONE_GROUP` or `ISOTROPIC_1G` ?

Reply:
I would vote no on `ISOTROPIC` because you can set this to be a pure absorber. To me `SIMPLE` is indicative of there only being absorption and scatter (i.e. no fission).

Reply:
ok

Reply:
I went with `MODEL` because it really is a model xs.  I'm ok with `SIMPLE` as well, so I've changed it to `SIMPLE_ONE_GROUP`.